### ISSUE TITLE
the cache time unit compatible

### ISSUE THREAD

**Issue by fly2006zhao** [andersao/l5-repository#711]
the laravel cache time unit is change form minute to seconds since Laravel 5.8
hope to fix it

**Comment by Jedliu** [comment]
Yes, found the same issue. Hope it will be fixed to avoid misunderstanding. Thanks,
